client/game/snake.py

```diff
@@ -0,0 +1,15 @@
+import random as rand
+from pygame.math import Vector2
+import game.visuals as vs
+import pygame 
+class SNAKE:
+    def __init__(self):
+
+        self.body = [Vector2(15,8),Vector2(16,8),Vector2(17,8)]
```

Comment:
Instead of using Vector2, lets use tuples. You can define them like [(15,8), (16,8), (17, 8)]